### ISSUE TITLE
NIST Quert wavelength limits not implemented property

### ISSUE THREAD

**Issue by arikaegan** [astropy/astroquery#2889]
The upper and lower wavelength limits are not being set correctly. 

in core.py, _args_to_payload, lines 94 and 95, the min and max wavelengths are set incorrectly. The dictionary key is currently set as
```
request_payload["low_wl"] = min_wav 
request_payload["upp_wl"] = max_wav
```
They keys should instead be "low_w" and "upp_w":
```
request_payload["low_w"] = min_wav
request_payload["upp_w"] = max_wav
```

**Comment by keflavich** [comment]
You're right.  I'm not sure if this is an upstream change or just an error, but in either case, would you be willing to submit a PR to fix the broken lines?
https://github.com/astropy/astroquery/blob/main/astroquery/nist/core.py#L91-L92

Here's an example query performed today:
https://physics.nist.gov/cgi-bin/ASD/lines1.pl?spectra=&output_type=0&low_w=500&upp_w=600&unit=1&submit=Retrieve+Data&de=0&plot_out=0&I_scale_type=1&format=0&line_out=0&en_unit=0&output=0&bibrefs=1&page_size=15&show_obs_wl=1&show_calc_wl=1&unc_out=1&order_out=0&max_low_enrg=&show_av=2&max_upp_enrg=&tsb_value=0&min_str=&A_out=0&intens_out=on&max_str=&allowed_out=1&forbid_out=1&min_accur=&min_intens=&conf_out=on&term_out=on&enrg_out=on&J_out=on
and you can see `low_w` and `upp_w` in the parameter list

**Comment by arikaegan** [comment]
Yes I can do that later this week.
